examples/03_connectivity/plot_signal_extraction.py

```diff
@@ -90,4 +94,53 @@
 plt.subplots_adjust(left=.01, bottom=.3, top=.99, right=.62)
 plt.suptitle('No confounds', size=27)
 
-plt.show()
+###############################################################################
+# Check the relevance of chosen confounds: The distribution of voxel-to-voxel
+# correlations should be tight and approximately centered to zero.
+
+#######################################################################
+# Compute voxel-wise time series with and without confounds removal, using
+# NiftiMasker.
+from nilearn.input_data import NiftiMasker
+brain_masker = NiftiMasker(memory='nilearn_cache', verbose=5)
```

Comment:
You probably chose `verbose=5` because of the masker used above but why did we chose that? `verbose=1` is enough I think.